test/testHelpers/datasets/datasetHelper.ts

```diff
@@ -0,0 +1,138 @@
+import { Dataset, DatasetVersionState } from '../../../src/datasets/domain/models/Dataset';
+import { NodeHtmlMarkdown } from 'node-html-markdown';
+
+const DATASET_CREATE_TIME_STR = '2023-05-15T08:21:01Z';
+const DATASET_UPDATE_TIME_STR = '2023-05-15T08:21:03Z';
+const DATASET_RELEASE_TIME_STR = '2023-05-15T08:21:03Z';
+
+const DATASET_HTML_DESCRIPTION =
+  '<div><h1 class="test-class-to-ignore">Title 1</h1><p>Test paragraph 1</p><p>Test paragraph 2</p><p>Hello world</p><h2>Title 2</h2><h3>Title 3</h3></div>';
+
+export const createDatasetModel = (): Dataset => {
+  return {
+    id: 1,
+    persistentId: 'doi:10.5072/FK2/HC6KTB',
+    versionId: 19,
+    versionInfo: {
+      majorNumber: 1,
+      minorNumber: 0,
+      state: DatasetVersionState.RELEASED,
+      createTime: new Date(DATASET_CREATE_TIME_STR),
+      lastUpdateTime: new Date(DATASET_UPDATE_TIME_STR),
+      releaseTime: new Date(DATASET_RELEASE_TIME_STR),
+    },
+    license: {
+      name: 'CC0 1.0',
+      uri: 'https://creativecommons.org/publicdomain/zero/1.0/',
+    },
+    metadataBlocks: [
+      {
+        name: 'citation',
+        fields: {
+          title: 'test',
```

Comment:
I was trying to get the Dataset title. Is this the dataset title? `dataset.metadataBlocks[0].fields.title` ? What if somehow the array is not ordered and the citation is not in index 0 but in any other index? wouldn't it be better to access the title through  `dataset.metadataBlocks.citation.fields.title` ?

Reply:
I spoke with Guillermo and the initial structure makes sense since a dataset may o may not have a citation, so we don't want to force the citation key since it's just another metadataBlock

We stick with `dataset.metadataBlocks[0].fields.title` and we'll find the `name: 'citation'` property in the frontend